2.1.1/buster/build-time-pip-constraints.txt

```
@@ -0,0 +1,499 @@
+# Constraint the version pip will install during docker build
+
+# Editable install with no version control (apache-airflow==2.1.0.dev0)
+APScheduler==3.6.3
+Authlib==0.15.3
+Babel==2.9.1
+Flask-AppBuilder==3.3.0
+Flask-Babel==1.0.0
+Flask-Bcrypt==0.7.1
+Flask-Caching==1.10.1
+Flask-JWT-Extended==3.25.1
+Flask-Login==0.4.1
+Flask-OAuthlib==0.9.5
+Flask-OpenID==1.2.5
+Flask-SQLAlchemy==2.5.1
+Flask-WTF==0.14.3
+Flask==1.1.2
+GitPython==3.1.14
+HeapDict==1.0.1
+JPype1==1.2.1
+JayDeBeApi==1.2.3
+Jinja2==2.11.3
+Mako==1.1.4
+Markdown==3.3.4
+MarkupSafe==1.1.1
+PyHive==0.6.4
+PyJWT==1.7.1
+PyNaCl==1.4.0
+PySmbClient==0.1.5
+PyYAML==5.4.1
+Pygments==2.9.0
+SQLAlchemy-JSONField==1.0.0
+SQLAlchemy-Utils==0.37.2
+SQLAlchemy==1.3.24
+Sphinx==3.4.3
+Unidecode==1.2.0
+WTForms==2.3.3
+Werkzeug==1.0.1
+adal==1.2.7
+aiohttp==3.7.4.post0
+alabaster==0.7.12
+alembic==1.6.2
+amqp==2.6.1
+analytics-python==1.2.9
+ansiwrap==0.8.4
+apache-airflow-providers-airbyte==1.0.0
+apache-airflow-providers-amazon==1.4.0
+apache-airflow-providers-apache-beam==2.0.0
+apache-airflow-providers-apache-cassandra==1.0.1
+apache-airflow-providers-apache-druid==1.1.0
+apache-airflow-providers-apache-hdfs==1.0.1
+apache-airflow-providers-apache-hive==1.0.3
+apache-airflow-providers-apache-kylin==1.0.1
+apache-airflow-providers-apache-livy==1.1.0
+apache-airflow-providers-apache-pig==1.0.1
+apache-airflow-providers-apache-pinot==1.0.1
+apache-airflow-providers-apache-spark==1.0.3
+apache-airflow-providers-apache-sqoop==1.0.1
+apache-airflow-providers-celery==1.0.1
+apache-airflow-providers-cloudant==1.0.1
+apache-airflow-providers-cncf-kubernetes==1.2.0
```

```suggestion
apache-airflow-providers-cncf-kubernetes==1!1.2.1
```

fixes our Istio issue -- for all AC release we will have to use this